src/components/Layout/Layout.css

```diff
@@ -3,3 +3,13 @@
 	display: flex;
 	flex-direction: column;
 }
+
+.content-container {
+    flex: 1; 
+    overflow-y: auto;
```

Comment:
Please use `height: 100%` instead of these two. With setting height to 100%, it will naturally adapt to the space availably assigned by the parent element, making everything connected and easier to control.

Reply:
Oh another great catch! Thank you and done!